env-vars.txt

```diff
@@ -0,0 +1,76 @@
+export OPERAND_IMAGE_APPLICATION_UI=quay.io/open-cluster-management/application-ui@sha256:8b1644073c07425dc9a457b3e628fbc6fbf31e0c89c42c51f10a2fcc7489d1f4
```

Comment:
So, I believe this file is actually published anytime we get the new image shas, whether it is using mocks or real images. I will go ahead and delete the specific example, but it will regenerate every time we prep the install of the operator